docs/data_management/lts/tutorial/individual_lts_tutorial.md

```diff
@@ -0,0 +1,113 @@
+
+### Creating Buckets
+
+Long Term Storage (LTS) services like Amazon S3 uses a specific data organization model based on **buckets** and **objects**. Think of buckets as folders that contain individual pieces of data called objects. Each bucket has a unique name. We have documentation about basic terminology on s3 storage system [here](../index.md/#terminology) and a naming guide for LTS buckets [here](../index.md/#avoiding-duplicate-names-for-buckets).
+
+Once you have complete `s3cmd` configuration, you can create new buckets in your individual LTS storage. To create a bucket use a `mb`(make bucket) command:
+
+```bash
+s3cmd mb s3://your_bucket_name
+```
+
+Please replace `your_bucket_name` with your desired name. This command creates a new bucket that you named `your_bucket_name` in your LTS storage using the currently configured `s3cmd` profile.
+
+The image below shows the LTS bucket created successfully after running the command.
+
+![image-bucket](../images/create-bucket.png)
+
+### Managing Buckets
+
+To manage a bucket, various commands can be used. Below are some common `s3cmd` commands to interact with your LTS bucket and its objects:
+
+- To **lists all buckets** accessible with your current `s3cmd` profile, use command: `s3cmd ls`.
+- To **lists all files in a bucket** accessible with your current `s3cmd` profile, use command: `s3cmd ls s3://your_bucket_name`.
+- To **upload a file**  named `file.txt` to a bucket, use a command: `s3cmd put file.txt s3://your_bucket_name/`.
+- To **download a file** a file named `file.txt` from a bucket, use a command:`s3cmd get s3://your_bucket_name/file.txt`.
+- To **delete/remove** a bucket, use command: `s3cmd rb s3://your_bucket_name`.
+
+You can find a variety of `s3cmd` commands in our documentation at [here](../../lts/interfaces.md/#s3cmd-commands) and on the [S3tools website](https://s3tools.org/usage). For quick reference, you can also use the `s3cmd --help` command to view available options directly in your terminal.
+
+If you are continuing in the same session with your **conda environment already activated**, you can directly use the `s3cmd` commands. If you are starting a new session or returning at a later date, make sure to load the Anaconda module and activate your conda environment before using `s3cmd`.
+
+<!-- markdownlint-disable MD046 -->
+!!! Note
+
+    An S3 bucket cannot be deleted unless it is completely empty. If the bucket contains any objects, `s3cmd` will return an error, like `S3 error: 409 (BucketNotEmpty)` when you attempt to delete it. To remove all objects within the bucket, use: `s3cmd del -r s3://your_bucket_name`. You can then remove the bucket itself with: `s3cmd rb s3://your_bucket_name`.
+<!-- markdownlint-disable MD046 -->
+
+<!-- markdownlint-disable MD046 -->
+!!! danger
+
+    Be cautious when deleting buckets, as this action is permanent and will remove all data within the bucket. Once data is deleted from LTS, it cannot be recovered.
```

Comment:
"Deleting objects and buckets cannot be undone. Once the delete command is entered, any data is lost permanently and cannot be restored."